sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCCommentsInMethodsCheck.java

```diff
@@ -39,15 +40,21 @@
      */
     public static final String MSG_KEY = "forbid.c.comments.in.the.method.body";
 
-    /**
-     * Set contains C style comments from current file.
-     */
-    private Set<Integer> clangComments;
+    /** AST to use for null checks because {@link Deque} doesn't allow null elements. */
+    private static final DetailAST NULL_AST = new DetailAST();
```

Comment:
`Deque` does not allow `null` elements to be placed inside it.
I originally wanted a Boolean to say in method or not, but I need to keep track of the current method AST for checking line numbers.

Making static class to hold the `null` AST seems pointless, so this was my workaround.